selfdrive/car/fingerprints.py

```diff
@@ -1,13 +1,15 @@
+from typing import Dict, List
 from openpilot.selfdrive.car.interfaces import get_interface_attr
 
+OfflineCanFingerprints = Dict[str, List[Dict[int, int]]]
```

Comment:
Why is this str? This doesn't look correct

Reply:
added this line, never fails when running test_can_fingerprints.py

`assert all([isinstance(k, int) and isinstance(v, int) for k, v in car_fingerprint.items()])`